.storybook/preview.js

```diff
@@ -19,7 +15,7 @@ export const parameters = {
     isToolshown: true,
   },
   docs: {
-    theme: themes.dark,
+    theme: themes.light,
   },
```

Comment:
Can we keep dark mode. Your render of the page shows cards with white text
![image](https://user-images.githubusercontent.com/17132317/136142453-941982cc-3179-4dcf-8a93-65a653960c82.png)

Reply:
Good catch. I'll fix it for this story.
I'm changing the theme to light for the docs tab of each story.
e.g. the docs of this story doesn't look so good.
https://councildataproject.org/cdp-frontend/?path=/docs/library-details-minutesitemslist--minutes-items-list

Reply:
Ahhhh interesting. Cool yea if the homepage is fixed. Happy to go back to light mode.